internal/core/build.sh

```diff
@@ -155,6 +155,8 @@ CMAKE_CMD="cmake \
 -DMILVUS_WITH_PROMETHEUS=${WITH_PROMETHEUS} \
 -DMILVUS_CUDA_ARCH=${CUDA_ARCH} \
 -DCUSTOM_THIRDPARTY_DOWNLOAD_PATH=${CUSTOM_THIRDPARTY_PATH} \
+-DMILVUS_ENABLE_PROFILING=${SUPPORT_PROFILING} \
```

Comment:
where will new variables be assigned value?

Reply:
if not , why add these options

Reply:
It is brought from: https://github.com/milvus-io/knowhere/blame/main/build.sh.

Reply:
so, the variable's default is missing?